src/main/java/org/craftercms/studio/impl/v1/service/deployment/PublishingManagerImpl.java

```diff
@@ -230,6 +231,14 @@ else if (StringUtils.equalsIgnoreCase(LIVE_ENVIRONMENT, item.getEnvironment()) |
                 itemMetadata.setLaunchDate(null);
                 objectMetadataManager.updateObjectMetadata(itemMetadata);
             }
+            if (ContentUtils.matchesPatterns(item.getPath(),
+                    Arrays.asList(studioConfiguration
```

Comment:
Is this going through a cached service?

Reply:
that is YAML configuration

Reply:
It is cached after we looked more deeply at it.